src/solver/type_getters.jl

```diff
@@ -57,6 +57,7 @@ function get_atmos(config::AtmosConfig, params)
     atmos = AtmosModel(;
         moisture_model,
         ozone,
+        co2 = PrescribedCO2(),
```

Comment:
Does it mean we always prescribe CO2? Could we add an option to set co2 as a constant? You can either use `input_vmr("carbon_monoxide_GM")`, or hard code the value 397.547.